lab1b/syntax.in

```diff
@@ -0,0 +1,22 @@
+program = "var" decllist ";" cmpdstmt ";"
+decllist = declaration ";"
+declaration = type IDENTIFIER ";" | type assignstmt ";"
```

Comment:
int number1 = readInt();
type assignstmt ";" does not match the line of code above

Reply:
int[] numbers = [1, 2, 2, 3];
type assignstmt ";" does not match the line of code above
[1, 2, 2, 3] is not an expression with the rules below

if _type_ may expand into _arraydecl_ and _assignstmt_ into _IDENTIFIER "=" expression ";"_ then _type assignstmt ";"_ may expand into _int abc [] abc = 4;;;;_